mne/decoding/classifier.py

```diff
@@ -453,3 +453,23 @@ def transform(self, epochs_data, y=None):
                                      picks=self.picks, n_jobs=self.n_jobs,
                                      copy=False, verbose=False)
         return epochs_data
+
+
+def compute_patterns(epochs, linear_model):
+    from sklearn.preprocessing import StandardScaler
+    labels = epochs.events[:, -1]
+    # concatenate channel x time data
+    X = epochs.get_data().reshape(len(epochs), -1)
+    # normalize the data
+    sc = StandardScaler()
+    X = sc.fit_transform(X)
```

Comment:
I think you should keep the fitting and transform stage separated. If one forgets to fit using your method, he/she could still want to look at the patterns.